_1327/polls/models.py

```diff
@@ -54,6 +54,13 @@ def generate_new_title(cls):
 		return _("New Poll")
 
 	@classmethod
+	def get_vote_permission(klass):
```

Comment:
please refactor. there is `def vote_permission_name(self)` just two methods below.